test/link.test.js

```diff
@@ -103,3 +103,15 @@ test("pass through attributes", () => {
   expect(vnode.attributes.href).toEqual("/path")
   expect(vnode.attributes.pass).toEqual("through")
 })
+
+test('custom onclick handler', () => {
```

Comment:
Run `npm run format`.

Reply:
<img width="387" height="306" src="https://user-images.githubusercontent.com/640669/38132810-5833a9ce-3415-11e8-8da3-a61e81e4d8d2.png">

Reply:
`npm run format -- 'test/**/*.js'`

The test directory is still not in the format config

Reply:
The script is not covering the test files, whoops. Could you add it?

Reply:
PR: https://github.com/hyperapp/router/pull/62

Reply:
💯